lizrd/scripts/run_exp_remotely.sh

```diff
@@ -18,14 +18,15 @@ run_grid_remotely() {
   host=$1
   config=$2
   session_name=$(date "+%Y_%m_%d_%H_%M_%S")
-
+  echo "Running grid search on $host with config $config"
 
   script="cd $base_dir && tmux new-session -d -s $session_name bash"
   script+="; tmux send-keys -t $session_name 'python3 -m lizrd.scripts.grid $config' C-m"
   script+="; tmux attach -t $session_name"
+  script+="; echo 'done'"
```

Comment:
Let's let future programmers know this line is black amgic

Reply:
done